BE/todo-list/src/main/java/team_16/todolist/domain/Board.java

```diff
@@ -0,0 +1,76 @@
+package team_16.todolist.domain;
+
+import org.springframework.data.annotation.Id;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+
+public class Board {
+
+    @Id
+    private Long id;
+
+    public String title;
+    private Set<Card> cards = new HashSet<>();
+
+    public Long getId() {
+        return id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public void setCards(Set<Card> cards) {
+        this.cards = cards;
+    }
+
+    public Board(String title) {
+        this.title = title;
+    }
+
+    public Card addCard(String title, String content) {
+        Card newCard = createCard(title, content);
+        cards.add(newCard);
+
+        return newCard;
+    }
+
+    private Card createCard(String title, String content) {
+        Card card = new Card(title, content);
+
+        return card;
+    }
+
+    public Set<Card> getCards() {
+        return cards;
+    }
+
+    public void deleteCard(Long cardId) {
+        getCard(cardId);
+        cards.remove(getCard(cardId));
+    }
+
+    public Card updateCard(Long cardId, String title, String content) {
+
+        return getCard(cardId).updateCard(title, content);
+    }
+
+    public Card getCard(Long cardId) {
+        Optional<Card> findCard = null;
+
+        for (Card card : cards) {
+            if (card.getId() == cardId) {
+                findCard = Optional.of(card);
+            }
+        }
+
+        return findCard.orElseThrow(IllegalArgumentException::new);
+    }
```

Comment:
stream을 사용하면 간단하게 표현할 수 있습니다.
```suggestion
       return cards.stream().filter(card → card.getId() == cardId).findOne().orElseThrow(IllegalArgumentException::new);
```

그리고, IllegalArgumentExcpetion 대신에 CustomException의 사용을 고려해봐주세요.

Reply:
줄바꿈은 안했는데, 줄바꿈을 해주면 더 가독성이 높아지겠죠?

Reply:
- 스트림 사용이 익숙치 않았는데 한번더 배웠습니다ㅜ
- 스트림 적용후 줄바꿈 추가했습니다
- 커스텀익셉션을 다음에 한번에 추가하도록 해보겠습니다